mmv1/third_party/terraform/tests/resource_compute_instance_group_manager_test.go.erb

```diff
@@ -630,6 +630,12 @@ resource "google_compute_instance_group_manager" "igm-update" {
     }
   }
 <% end -%>
+
+<% unless version == "ga" -%>
```

Comment:
Is this field updatable? If yes can we make sure to test that?

Reply:
Thank for pointing, updated tests